index/src/main/scala/org/dbpedia/spotlight/db/CreateVectorStore.scala

```diff
@@ -0,0 +1,39 @@
+package org.dbpedia.spotlight.db
```

Comment:
Is this necessary for this PR ? I thought somehow the functionality was written in the previous one ?